### ISSUE TITLE
Support multiple feature bands

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/dotnet#1011]
**Scenario:**
* Multiple feature bands (e.g. 6.0.1xx and 6.0.2xx) are released on the same day.  
* Source-build today only supports building each release tarball separately which each produce a copy of the runtime (logically identical).
* Source-build tooling should support building and releasing both but with a shared runtime just like the Microsoft release.

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.

**Comment by tmds** [comment]
Every feature band involves some work.
If there is enough compatibility, having the latest would be preferable.

I guess one of the driving reasons for feature bands is to ensure compatibility for visual studio.
I think this is less important for source-built .NET.

cc @omajid

**Comment by omajid** [comment]
> I guess one of the driving reasons for feature bands is to ensure compatibility for visual studio. I think this is less important for source-built .NET.

Agreed. I think we care more about *backwards compat* for source-build users. We would be okay with new features, but it's backwards compat that have a large impact.

Red Hat ships any feature band upgrades as monthly updates. That means customers will upgrade to new feature bands (without any choice) to get CVE fixes. Any backward incompatibility in the feature band has a risk of breaking users building .NET applications, and is almost too risky for the users/customers Red Hat cares about.

**Comment by mmitche** [comment]
@mthalman Is there work left here?

**Comment by mthalman** [comment]
Yes, the remaining 3 open sub-issues are still relevant.
